packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -230,6 +273,7 @@ class _ScaffoldLayout extends MultiChildLayoutDelegate {
         fabY = math.min(fabY, contentBottom - snackBarSize.height - fabSize.height - _kFloatingActionButtonMargin);
       if (bottomSheetSize.height > 0.0)
         fabY = math.min(fabY, contentBottom - bottomSheetSize.height - fabSize.height / 2.0);
+      fabY += 30.0;
```

Comment:
?

Reply:
:open_mouth:
That what I did to test an app with some overlap